loader/loader.go

```diff
@@ -103,7 +103,11 @@ type FileLoader interface {
 	WithCapabilities(*ast.Capabilities) FileLoader
 	WithJSONOptions(*astJSON.Options) FileLoader
 
+	// WithRegoV1Compatible
+	// Deprecated: use WithRegoVersion instead
```

Comment:
I think it's unlikely many users are using this already, so we can probably remove the `WithRegoV1Compatible` func now.